src/domain/entities/risk-assessment/__tests__/RiskAssessmentGrading.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { describe, expect, it } from "vitest";
+import {
+    HighCapacity,
+    HighGeographicalSpread,
+    HighPopulationAtRisk,
+    HighWeightedOption,
+    LowCapacity,
+    LowGeographicalSpread,
+    LowPopulationAtRisk,
+    LowWeightedOption,
+    MediumCapacity,
+    MediumGeographicalSpread,
+    MediumPopulationAtRisk,
+    MediumWeightedOption,
+    RiskAssessmentGrading,
+} from "../RiskAssessmentGrading";
+
+describe("RiskAssessmentGrading", () => {
+    it("should be Grade1 if total weight is less than or equal to 7", () => {
+        const riskAssessmentGrading = RiskAssessmentGrading.create({
+            id: "1",
+            lastUpdated: new Date(),
+            populationAtRisk: LowPopulationAtRisk,
+            attackRate: LowWeightedOption,
+            geographicalSpread: LowGeographicalSpread,
+            complexity: LowWeightedOption,
+            capacity: LowCapacity,
+            reputationalRisk: LowWeightedOption,
+            severity: LowWeightedOption,
+        });
+        const grade = riskAssessmentGrading.getGrade().value.data;
```

Comment:
Not sure why `either.value` is public, looks like internal state to me.

Proposal: `const value = either.getOrThrow("Error Message")`, and you won't need any if later.

Reply:
added and used  Either.getOrThrow as suggested. 
The if in next line is because RiskAssessmentGrading.getTranslatedLabel does not accept undefined as a parameter, only valid type can be passed as parameter.